### ISSUE TITLE
The test results only can be reproduced with the true test mask (not consistent with the paper)

### ISSUE THREAD

**Issue by zysxmu** [hhqweasd/G2R-ShadowNet#10]
Hi,
I have used the pre-trained models, but I found that the paper's test performances only can be reproduced with the true test mask.
That is, use these true mask in 'ISTD_Dataset/test/test_B/'.
<img width="364" alt="image" src="https://user-images.githubusercontent.com/95413091/159832343-243d035c-110e-4242-91b2-c384d8a895b4.png">

However, if use the detected mask: test_mask_bdrar.zip provided by this repo. The results are:
<img width="360" alt="image" src="https://user-images.githubusercontent.com/95413091/159832854-ff4568d1-8691-4127-9dcc-bec598255114.png">

While the paper says the results are obtained with the detected mask instead of real mask.
